EIPS/eip-4812.md

```diff
@@ -0,0 +1,45 @@
+---
+eip: 4812
+title: Exodus
+description: Exodus allows to create blockchain networks with accounts and transactions.
+author: Antoine Toulmé (@atoulme), Chaals Nevile (@chaals)
+discussions-to: <https://entethalliance.org/contact>, For EEA members only (requires a patent commitment): <https://github.com/entethalliance/testing-wg/issues> 
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-11-09
+---
+
+## Abstract
+This document specifies how to generate a blockchain environment from a file.
+
+The environment is created based on a chain configuration, that can include initial balances, and transactions to be processed.
+
+## Motivation
+During testing of distributed systems, we need to load blockchain data and perform some operations.
+
+There is currently no standard in place to codify how to ingest blockchain data. The current solutions involve directly ingesting RLP-encoded data, and that is effectively impossible to read directly which makes it hard to reason about tests.
+
+We aim to facilitate testing and formulation of tests through an explicit format developers can use to represent contracts and transactions in detail, as well as the expected outcomes.
+
+## Specification
```

Comment:
No specification?

Reply:
The spec was a bit stuck with our working group. I should be able to get it out.

Reply:
Linked to specification.